src/core/example_settings.py

```diff
@@ -395,3 +395,26 @@ def filter(self, record):
 WORKFLOW_PLUGINS = {}
 
 SILENT_IMPORT_CACHE = False
+
+# Development Overrides
+if DEBUG:
+    EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+    for t in TEMPLATES:
+        t["OPTIONS"]["string_if_invalid"] = "Invalid variable: %s!!"
+
+# Testing Overrides
+if IN_TEST_RUNNER:
```

Comment:
Can you clarify what this does? If running `manage.py test` skip migrations?

Reply:
Yes, Having to wait for all the migrations on every test run is quite a pain